front/src/applications/editor/tools/selection/components.tsx

```diff
@@ -109,6 +110,28 @@ export const SelectionLeftPanel = () => {
     EditorContext
   ) as ExtendedEditorContextType<SelectionState>;
   const { selection } = state;
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    const param = searchParams.get('selection');
+    // Remove the select param
+    if (selection.length === 0 && param) {
+      searchParams.delete('selection');
+      // Replace to avoid the user to be able to press "back" or "forward" on
+      // the browser because the selection wouldn't change
+      setSearchParams(searchParams, { replace: true });
+      // If selection is too large, we get an error because the url is too long
+    } else if (selection.length < 100) {
```

Comment:
If you have a selection : 
if selection is less than 100 then update the slection param in url 
else, remove the param in the url , otherwise you can kept an old param in the url.